azure_monitor/src/azure_monitor/sdk/auto_collection/metrics_span_processor.py

```diff
@@ -13,49 +12,21 @@
 
 class AzureMetricsSpanProcessor(SpanProcessor):
     """AzureMetricsSpanProcessor is an implementation of `SpanProcessor` used
-    to generate Azure specific metrics, including dependencies/requests rate, average duration
-    and failed dependencies/requests.
+    to generate documents for Live Metrics.
     """
 
     def __init__(self):
         self.is_collecting_documents = False
         self.documents = collections.deque()
-        self.request_count = 0
-        self.dependency_count = 0
-        self.failed_request_count = 0
-        self.failed_dependency_count = 0
-        self.request_duration = 0
-        self.dependency_duration = 0
 
     def on_start(self, span: Span) -> None:
         pass
 
     def on_end(self, span: Span) -> None:
         try:
-            if span.kind == SpanKind.SERVER:
-                self.request_count = self.request_count + 1
-                duration = (
-                    span.end_time - span.start_time
-                ) / 1000000  # Convert to milliseconds
-                self.request_duration = self.request_duration + duration
+            if self.is_collecting_documents:
                 if not span.status.is_ok:
-                    self.failed_request_count = self.failed_request_count + 1
-                    if self.is_collecting_documents:
-                        self.documents.append(convert_span_to_envelope(span))
-
-            elif span.kind == SpanKind.CLIENT:
-                self.dependency_count = self.dependency_count + 1
-                duration = (
-                    span.end_time - span.start_time
-                ) / 1000000  # Convert to milliseconds
-                self.dependency_duration = self.dependency_duration + duration
-                if not span.status.is_ok:
-                    self.failed_dependency_count = (
-                        self.failed_dependency_count + 1
-                    )
-                    if self.is_collecting_documents:
-                        self.documents.append(convert_span_to_envelope(span))
-
+                    self.documents.append(convert_span_to_envelope(span))
         # pylint: disable=broad-except
         except Exception:
             logger.exception("Exception while processing Span.")
```

Comment:
For now we should be consistent and use `logger.warning`

Reply:
I think this is still here?

Reply:
Just realized this was the case in tons of different places, updated all auto_collection ones now, we still do logger.exception when exporter fails, not sure if that one is by design